h2o-core/src/main/java/water/Weaver.java

```diff
@@ -54,6 +54,8 @@
     } catch( NotFoundException nfe ) { throw new RuntimeException(nfe); }
   }
 
+  public static ClassPool getPool() { return _pool; }
```

Comment:
Seems like you have a specific use case, I think it might be a good idea to document why is the pool exposed to the world (someone might remove this function since it is not used in code base of H2O).